lib/task_helpers/exports.rb

```diff
@@ -0,0 +1,35 @@
+module TaskHelpers
+  class Exports
+    def self.new_filename(filename, keep_spaces = false)
+      new_filename = filename
+      new_filename = new_filename.gsub(%r{[ ]}, '_') unless keep_spaces
+      new_filename = new_filename.gsub(%r{[/]}, 'slash')
+      new_filename.gsub(%r{[|]}, 'pipe')
```

Comment:
I missed that when I read the doc page the first time. I've changed to use a hash.